suites/nautilus/rgw/tier-2_rgw_test-bucket-lifecycle-rgw-multisite-primary-to-secondary.yaml

```diff
@@ -0,0 +1,122 @@
+# Below are the multi-site test scenarios run on the secondary and verified the sync/io on the primary
```

Comment:
Would it be possible to rename the file to `tier-2_rgw_multisite_test-bucket-lifecycle-primary-to-secondary`? This would allow us to keep the file naming consistent...

Reply:
Done.